### DISCUSSION TITLE
I get an error message when trying to create a item.

### DISCUSSION THREAD

**Discussion by ImprovedFaund** [FabricMC/fabric-api#3023] [Mod Dev Support]
When I try creating an item I get this error. The method register(DefaultedRegistry<Item>, Identifier, Item) is undefined for the type Registry.

The Code:
package net.wakanda.mod;

import org.apache.http.config.Registry;

import net.fabricmc.api.ModInitializer;
import net.fabricmc.fabric.api.item.v1.FabricItemSettings;
import net.minecraft.item.Item;
import net.minecraft.item.ItemGroup;
import net.minecraft.registry.Registries;
import net.minecraft.util.Identifier;
import net.minecraft.world.IdCountsState;

public class WakandaMod implements ModInitializer {

    public static final Item FABRIC_ITEM = new Item(new FabricItemSettings());
    
    @Override
    public void onInitialize() {
        Registry.register(Registries.ITEM, new Identifier("wakandamod", "fabric_item"), FABRIC_ITEM);
    }
    
}

**Comment by warjort** [comment]
You are trying to use the Registry class from apache's http client code:

import org.apache.http.config.Registry;

You want the minecraft Registry class:
https://github.com/FabricMC/yarn/blob/23w16a/mappings/net/minecraft/registry/Registry.mapping
